imagersite/imager_images/templates/imager_images/photo_gallery.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'imagersite/base.html' %}
+{% load thumbnail %}
+{%  block body %}
+    <h3>{{ user.username }} Photos</h3>
+
+    <ul class="thumbnail">
+        {% for photo in photos %}
+        <li>
+            {% thumbnail photo.photo "100x100" crop="center" as im %}
+                <img src="{{ im.url }}" width="{{ im.width }}" height="{{ im.height }}">
+            {% endthumbnail %}
+            <a href="{% url 'imager_images:photo' %}">{{photo.title}}</a>
```

Comment:
You are missing the id for url `<a href="{% url 'imager_images:photo' %}">{{photo.title}}</a>` should be `<a href="{% url 'imager_images:photo' photo.id %}">{{photo.title}}</a>`